playbook-website/app/javascript/components/Website/src/components/CategoryTitle/styles.scss

```diff
@@ -1,6 +1,6 @@
-@import "playbook-ui/dist/tokens/screen_sizes";
-@import "playbook-ui/dist/tokens/animation-curves";
-@import "playbook-ui/dist/tokens/transition";
+@import "playbook-tokens/screen_sizes";
+@import "playbook-tokens/animation-curves";
+@import "playbook-tokens/transition";
```

Comment:
You will notice this happening in a lot of the SCSS files. This way, we can change the alias without having to touch every single file. #futureproofing